src/screens/OtherProfile/elements/BioAndChat.js

```diff
@@ -36,6 +38,19 @@ const BioAndChat = (props) => {
     setDmChat('');
   }, [isSignedMessageEnabled]);
 
+  React.useEffect(() => {
+    const unsubscribe = navigation.addListener('blur', () => {
+      setTimeout(() => {
+        setDmChat('');
+        setLoadingSendDM(false);
+      }, 500);
+    });
+
+    return () => {
+      unsubscribe();
+    };
+  }, []);
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

This is a good use of the `useEffect` hook to clean up state when the component is blurred. However, using `setTimeout` without clearing it in the cleanup function can lead to potential memory leaks if the component unmounts before the timeout finishes. Consider clearing the timeout in the cleanup function.
```diff
  React.useEffect(() => {
    const unsubscribe = navigation.addListener('blur', () => {
      const timeoutId = setTimeout(() => {
        setDmChat('');
        setLoadingSendDM(false);
      }, 500);
+     return () => clearTimeout(timeoutId);
    });

    return () => {
      unsubscribe();
    };
  }, []);
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->